### PR TITLE
Sigma queries - defense evasion by tampering with svchost; recently released GALLIUM activity group IOCs

### PR DESCRIPTION
**Pull request by timbMSFT**
-- terminating threads in a svchost process (InvokePhantom uses this technique to disable windows event logging)
-- GALLIUM threat intel IOCs in recent MSTIC blog/release.

### CODE REVIEW

File: rules/apt/apt_gallium.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+falsepositives:
+    - unknown
+level: high
+---
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    exec_selection:
+        Hashes:
```
Comment (Neo23x0) [inline rules/apt/apt_gallium.yml]: @thomaspatzke : do we use the `Hashes` field from Sysmon or do we always require the values to be split up into their respective hash type?

File: rules/apt/apt_gallium.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            - '*1c6452026c56efd2c94cea7e0f671eb55515edb0*'
+            - '*c6b41d3afdcdcaf9f442bbe772f5da871801fd5a*'
+            - '*4923d460e22fbbf165bbbaba168e5a46b8157d9f*'
+            - '*f201504bd96e81d0d350c3a8332593ee1c9e09de*'
+            - '*ddd2db1127632a2a52943a2fe516a2e7d05d70d2*'
+    condition: exec_selection
+---
+logsource:
+    product: windows
+    service: dns-server
```
Comment (Neo23x0) [inline rules/apt/apt_gallium.yml]: We don't have many DNS rules. This may be the next candidate for a generic category. @thomaspatzke

Comment (timbMSFT) [reply rules/apt/apt_gallium.yml]: that would be great -- happy to talk more about this offline. I'm experimenting with some ala.py backend changes to better support DNS there. It is a primary category for sharing threat intel IOCs. We also have a number of DNS-related TTP rules in the AzureSentinel GitHub that would be good candidates to have Sigma versions also.

File: rules/apt/apt_gallium.yml (hunk 1 above)
Comment (timbMSFT) [reply rules/apt/apt_gallium.yml]: My reference for using Hashes was: https://github.com/Neo23x0/sigma/wiki/Taxonomy

File: rules/apt/apt_gallium.yml (hunk 2 above)
Comment (timbMSFT) [reply rules/apt/apt_gallium.yml]: .... but we could close on this PR (rules) and work on generic dns support separately?

File: rules/apt/apt_gallium.yml (hunk 1 above)
Comment (thomaspatzke) [reply rules/apt/apt_gallium.yml]: We use Hashes, as this is how the hashes appear in their events.

File: rules/apt/apt_gallium.yml (hunk 2 above)
Comment (thomaspatzke) [reply rules/apt/apt_gallium.yml]: Yes, as well as generic descriptions of registry changes, which are ranked after process creations. Lets do it as @timbMSFT suggested and merge this rule. I have some rule cleanup of inconsistent on my agenda anyways.

**Pull request merged**